.github/workflows/integration-test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [20.x]
```

Comment:
Here we could have tests running on `[18.x, 20.x]`, but at first I see no benefit in this extra round of tests on a version that's already in maintenance mode.

Reply:
Also, in case people always use the Docker image to run the wallet, there is a guarantee that we only need to support a specific version.